app/single_resource/views.py

```diff
@@ -20,19 +21,30 @@ def index():
 @login_required
 def create():
     """Add a resource."""
+    descriptors = Descriptor.query.all()
+    for descriptor in descriptors:
+        if (descriptor.values):  # fields for option descriptors
```

Comment:
Nit: comments be on the line before the relevant line of code (and preferably capitalized and finished with a period).